<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_calib</class>
 <widget class="QDialog" name="Dial_calib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>600</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="StepClk">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>93</width>
     <height>63</height>
    </rect>
   </property>
   <property name="text">
    <string>StepClock</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="FrontEnd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PhaseAdc">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PhaseAdc</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PhaseDrsAdc">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CalibVco">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>250</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CalibVco</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CopyToEeprom">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>290</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy to EEprom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectClock">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cycle Power Cdce62005 </string>
   </property>
  </widget>
  <widget class="QComboBox" name="ShiftClockPllFpga">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>103</width>
     <height>26</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>ALL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M Counter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C0 output  120 Mhz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C1 output  not use</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C2 output   clkadc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C3 output  120 Mhz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C4 output  clk to adc</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>nÂ° ASM</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Choice Pattern</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>140</width>
     <height>142</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="ChoicePattern">
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0xFFF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0xAAA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0xFFF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0xAAA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x03F</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0xA33</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0x000</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StartCalib">
      <property name="text">
       <string>Start Pattern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopCalib">
      <property name="text">
       <string>Stop Pattern</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift Clock DRS - ADC</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>180</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="title">
    <string>Cdce 62005</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string>Phase pll Adc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>StatPll</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StatPll">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>75</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="Ch8">
      <property name="text">
       <string>Ch 8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch9">
      <property name="text">
       <string>Ch 9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch10">
      <property name="text">
       <string>Ch 10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch11">
      <property name="text">
       <string>Ch 11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch12">
      <property name="text">
       <string>Ch 12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch13">
      <property name="text">
       <string>Ch 13</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch14">
      <property name="text">
       <string>Ch 14</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch15">
      <property name="text">
       <string>Ch 15</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>65</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="Ch0">
      <property name="text">
       <string>Ch 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch1">
      <property name="text">
       <string>Ch 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch2">
      <property name="text">
       <string>Ch 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch3">
      <property name="text">
       <string>Ch 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch4">
      <property name="text">
       <string>Ch 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch5">
      <property name="text">
       <string>Ch 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch6">
      <property name="text">
       <string>Ch 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch7">
      <property name="text">
       <string>Ch 7</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>75</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="Ch16">
      <property name="text">
       <string>Ch 16</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch17">
      <property name="text">
       <string>Ch 17</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch18">
      <property name="text">
       <string>Ch 18</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch19">
      <property name="text">
       <string>Ch 19</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch20">
      <property name="text">
       <string>Ch 20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch21">
      <property name="text">
       <string>Ch 21</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch22">
      <property name="text">
       <string>Ch 22</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Ch23">
      <property name="text">
       <string>Ch 23</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>371</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift Clock Deseriarlize ADC </string>
   </property>
  </widget>
  <widget class="QComboBox" name="DWriteDenable">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>410</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dwrite only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Denable only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dwrite and Denable</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="StartDaq">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>560</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Daq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Monitoring">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Monitoring</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NbStep">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nb Step</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ModeDaq">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>490</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mode Thor 3 mezzanines</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza0-Mezza1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza2-Mezza0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza2-Mezza1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And 3 Mezza</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And Mezza 0-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And Mezz 1-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode ASM Pied</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="FreqPied">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_freqPied">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>530</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq Pied</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AndOrQuartet">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>450</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>OR   Quartet By Mezza </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AND Quartet By Mezza</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label_freqPied_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>530</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*256*1Âµs</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ValFreq">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculFreq">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcul Freq</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>330</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Mhz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="allCurves">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>102</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>all curves</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ValFreq_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ValFreq_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>360</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Mhz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>390</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Mhz</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="CalculFreqBar">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>380</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="OneSec">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>560</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;1 sec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Autocalib">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>AutoCalib</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dial_calib</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dial_calib</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
